Editor/HPUIInteractorPillEditor.cs

```diff
@@ -0,0 +1,65 @@
+using UnityEngine;
+using UnityEditor;
+using ubco.ovilab.HPUI.Interaction;
+
+namespace ubco.ovilab.HPUI.Editor
+{
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(HPUIInteractorPill), true)]
+    public class HPUIInteractorPillEditor : UnityEditor.Editor
+    {
+        private HPUIInteractorPill t;
+        private bool generating = false;
+        private float maxAngle = 90;
```

Comment:
Change this to 180

Reply:
doesn't line up with full range, that would need to be edited first

Reply:
huh wdym. this is a private variable. it doesnt matter?

Reply:
aite done